samples/outbox/sqltransport-nhpersistence-ef/SqlServer_3/Receiver/OrderSubmittedHandler.cs

```diff
@@ -17,7 +19,7 @@ public class OrderSubmittedHandler :
 
         var storageContext = context.SynchronizedStorageSession.Session();
 
-        var dbConnection = storageContext.Connection;
+        var dbConnection = (SqlConnection) storageContext.Connection;
         using (var receiverDataContext = new ReceiverDataContext(dbConnection))
         {
             var dbTransaction = ExtractTransactionFromSession(storageContext);
```

Comment:
@MarcinHoppe @SimonCropp I don't see `Transaction` property there. I think we used to have it in NSB 5. The purpose of this dummy commands is to extract the DbConnection from NHibernate's transaction object. It is a hack.